channels/vn.m3u

```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
+https://truyenhinh.vnptvas.vn/live.m3u8?c=vstv208&q=high&type=tv&token=1111&gcId=1532&userId=&deviceId=&deviceType=&location=NA&pkg=pkg11
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Oh Muang Lao TV
+https://livefta.malimarcdn.com/ftaedge00/ohmuanglao.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Tea TV
+https://livefta.malimarcdn.com/ftaedge00/teatv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Good Idea TV
+https://livefta.malimarcdn.com/ftaedge00/gooditv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Muslim TV
+https://livefta.malimarcdn.com/ftaedge00/ddtvlao.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Brian TV
+https://livefta.malimarcdn.com/ftaedge00/briantv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Ning TV
+https://livefta.malimarcdn.com/ftaedge00/ningtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Air TV
+https://livefta.malimarcdn.com/ftaedge00/airtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Dao Lane Xang
+https://livefta.malimarcdn.com/ftaedge00/khaomwungmai.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Oy Lao TV
+https://livefta.malimarcdn.com/ftaedge00/oylaotv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vati Lao TV
+https://livefta.malimarcdn.com/ftaedge00/vatilaotv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Houng Fa
+https://livefta.malimarcdn.com/ftaedge00/houngfa.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Dhammasapha TV
+https://livefta.malimarcdn.com/ftaedge00/dhammasapha.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Khomsanh TV
+https://livefta.malimarcdn.com/ftaedge00/lstv2.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Heritage Foundation TV
+https://livefta.malimarcdn.com/ftaedge00/laoheritagetv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Supreme Master TV
+https://lbs-us1.suprememastertv.com/720p.m3u8
```

Comment:
Language of this channel is not only Vietnamese, I see some English parts.
![vlcsnap-2021-05-31-17h29m40s270](https://user-images.githubusercontent.com/30985701/120215256-ca6b0a80-c235-11eb-9cf8-866dec7a3869.png)